packages/components/form/src/form-item.vue

```diff
@@ -119,83 +134,50 @@ export default defineComponent({
       }
       return false
     })
-
-    let initialValue = undefined
-
-    watch(
-      () => props.error,
-      (val) => {
-        validateMessage.value = val
-        validateState.value = val ? 'error' : ''
-      },
-      {
-        immediate: true,
-      }
-    )
-    watch(
-      () => props.validateStatus,
-      (val) => {
-        validateState.value = val
-      }
-    )
-
-    const labelFor = computed(() => props.for || props.prop)
-    const labelStyle = computed(() => {
-      const ret: CSSProperties = {}
-      if (elForm.labelPosition === 'top') return ret
-      const labelWidth = addUnit(props.labelWidth || elForm.labelWidth)
-      if (labelWidth) {
-        ret.width = labelWidth
-      }
-      return ret
-    })
-    const contentStyle = computed(() => {
-      const ret: CSSProperties = {}
-      if (elForm.labelPosition === 'top' || elForm.inline) {
-        return ret
-      }
-      if (!props.label && !props.labelWidth && isNested.value) {
-        return ret
-      }
-      const labelWidth = addUnit(props.labelWidth || elForm.labelWidth)
-      if (!props.label && !slots.label) {
-        ret.marginLeft = labelWidth
-      }
-      return ret
-    })
     const fieldValue = computed(() => {
-      const model = elForm.model
-      if (!model || !props.prop) {
-        return
-      }
+      const model = formContext?.model
+      if (!model || !props.prop) return
```

Comment:
Should we skip `{}` here?

Reply:
I think it might just be a code style. Both of them are acceptable.